docs/lib/datastore/fragments/android/sync/50-selectiveSync.md

```diff
@@ -0,0 +1,284 @@
+## Selectively syncing a subset of your data
+
+By default, DataStore downloads the entire contents of your cloud data source to your local device.
+The max number of records that will be stored is configurable [here](~/lib/datastore/conflict.md).
+
+You can utilize selective sync to only persist a subset of your data instead.
+
+Selective sync works by applying predicates to the base and delta sync queries, as well as to incoming subscriptions.
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.addPlugin(new AWSDataStorePlugin(DataStoreConfiguration.builder()
+    .syncExpression(Post.class, () -> Post.RATING.gt(5))
+    .syncExpression(Comment.class, () -> Comment.STATUS.eq("active"))
+    .build()));
+Amplify.configure(context);
+```
+
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+Amplify.addPlugin(AWSDataStorePlugin(DataStoreConfiguration.builder()
+    .syncExpression(Post::class.java) { Post.RATING.gt(5) }
+    .syncExpression(Comment::class.java) { Comment.STATUS.eq("active") }
+    .build()))
+Amplify.configure(context)
```

Comment:
Do we need to include the call to `configure`, you think?

Reply:
I just removed it in the latest commit - better to be brief I think!